src/planwise/model/scenarios.clj

```diff
@@ -6,10 +6,15 @@
 (s/def ::provider-id string?)
 (s/def ::location map?)
 
-(s/def :create-provider/action #{"create-provider"})
+(s/def ::action #{"create-provider"})
+(s/def ::initial boolean?)
 
-(s/def ::create-provider-change
-  (s/keys :req-un [:create-provider/action ::investment ::capacity ::provider-id ::location]))
+(s/def ::create-provider
+  (s/keys :req-un [::action ::investment ::capacity ::provider-id ::location]))
+
+(s/def ::initial-provider
+  (s/keys :req-un [::initial ::provider-id ::capacity ::location]))
 
 (s/def ::change-set
-  (s/coll-of ::create-provider-change))
+  (s/coll-of (s/or :create-provider ::create-provider
```

Comment:
I think this is a leftover from a previous implementation. AFAICS, you are not putting initial providers in the `changeset` (which is good btw), but in a separate `:providers` key.